packages/p2p/src/constants/account-status-codes.ts

```diff
@@ -0,0 +1,11 @@
+export const identity_status_codes = {
+    PENDING: 'pending',
+    REJECTED: 'rejected',
+    VERIFIED: 'verified',
+} as const;
+
+export const document_status_codes = {
+    PENDING: 'pending',
+    REJECTED: 'rejected',
+    VERIFIED: 'verified',
+} as const;
```

Comment:
i know these are different, but they are the same statuses, can we not just combine these?